go/templates/index.tmpl

```diff
@@ -285,24 +267,28 @@
         $('.loading-indicator').hide();
       });
     } else {
-      var handler = Plaid.create({
-        ...linkHandlerCommonOptions,
-        // webhook: 'https://your-domain.tld/plaid-webhook',
-        onSuccess: function(public_token) {
-          $.post('/set_access_token', {
-            public_token: public_token
-          }, function(data) {
-            $('#container').fadeOut('fast', function() {
-              $('#item_id').text(data.item_id);
-              $('#access_token').text(data.access_token);
-              $('#intro').hide();
-              $('#app, #steps').fadeIn('slow');
+      var handler = null;
+      $.post('/create_link_token', {}, function(data){
+          localStorage.setItem('link_token', data.link_token);
+          handler = Plaid.create({
+          token: data.link_token,
```

Comment:
nit: additional indent here for the `Plaid.create({})` object? 👾

Reply:
good catch